src/shared/mount-util.c

```diff
@@ -856,6 +948,17 @@ static int mount_in_namespace(
         if (S_ISLNK(st.st_mode)) /* This shouldn't really happen, given that we just chased the symlinks above, but let's better be safe… */
                 return log_debug_errno(SYNTHETIC_ERRNO(EOPNOTSUPP), "Source directory %s can't be a symbolic link", src);
 
+        if (!is_image && ms_nosymfollow_supported() > 0) /* Shortcut if we can use the new mount API */
+                return mount_in_namespace_new(chased_src_path,
```

Comment:
we usually add a line break after the `(`

Reply:
I'd prefer if we'd swap this around btw. i.e. that mount_in_namespace() is the one using the new mount option. And that the existing mount_in_namespace() gets renamed to mount_in_namespace_compat() or mount_in_namespace_legacy() or so. i.e. always put the new stuff front and center and the old stuff hide away

Reply:
if you swap this around you can drop the explicit feature check btw, just dispatch to the legacy call when you see ERRNO_IS_NOT_SUPPORTED() from open_tree()